docs/list.yaml

```diff
@@ -50,6 +50,42 @@ PHP:
       description: Adds CORS headers support in your Laravel application
       url: https://github.com/barryvdh/laravel-cors
       tags: ['Laravel', 'CORS']
+    barryvdh/laravel-debugbar:
+      description: Laravel Debugbar (Integrates PHP Debug Bar)
+      url: https://github.com/barryvdh/laravel-debugbar
+      tags: ['Laravel', 'Debugging']
+    jenssegers/mongodb:
+      description: A MongoDB based Eloquent model and Query builder for Laravel (Moloquent)
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Database', 'Moloquent']
+    robmorgan/phinx:
+      description: Simple PHP Database Migrations
+      url: https://github.com/cakephp/phinx
+      tags: ['Migrations']
+    phpoffice/phpspreadsheet:
+      description: A pure PHP library for reading and writing spreadsheet files
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Office']
+    spatie/laravel-activitylog:
+      description: Log activity inside your Laravel app https://docs.spatie.be/laravel-activitylog
+      url: https://github.com/spatie/laravel-activitylog
+      tags: ['Laravel', 'Activity Log', 'Logging']
+    vinkla/laravel-hashids:
+      description: A Hashids bridge for Laravel (generate short unique ids from integers)
+      url: https://github.com/vinkla/laravel-hashids
+      tags: ['Laravel', 'Hashids']
+    rap2hpoutre/laravel-log-viewer:
+      description: Log Viewer for Laravel 5 (compatible with 4.2 too) and Lumen
+      url: https://github.com/rap2hpoutre/laravel-log-viewer
+      tags: ['Laravel', 'Logging', 'Log Viewer']
```

Comment:
What the difference between  `Logging` and `Log Viewer` ?

Reply:
Difference between them:
rap2hpoutre/laravel-log-viewer
- The UI is simpler than (https://cloud.githubusercontent.com/assets/1575946/5243642/8a00b83a-7946-11e4-8bad-5c705f328bcc.png)
- Case-insensitive searching log entries
- Support searching in both messages and contexts
- Support sorting log entries by logged time, log level, message, ...
- Read all ".log" files in storage/logs

arcanedev/log-viewer
- The UI looks more beautiful than (https://raw.githubusercontent.com/ARCANEDEV/LogViewer/master/_screenshots/1-dashboard.jpg)
- There is dashboard page, show statistics info of log files
- Support filter by Log Level (info, debug, ...)
- Support searching in messages only
- Case-sensitive searching log entries
- Could specified the logs storage path
- Read all files that name match to log files pattern configuration (Ex: prefix: 'laravel-', extension: '.log')

Reply:
IMHO, "Logging" is a larger concept than "Log Viewer"
- monolog/monolog -> Log writer (or logger)
- rap2hpoutre/laravel-log-viewer -> Log viewer
so when I search Logging, I want to see both above packages in results.

Reply:
I think, ```Logging``` will be enough. I have doubts, that we will have hundreds of logging related sources, maybe 10 or 20. And I have doubts, that we will have more, than 1-2 sources, that can be marked as ```Log Viewer``` or ```Log Writer```. If we will create a new tag for each case, we will very quickly finish with tags cloud with hundreds of minoritary tags, and it will be hard to find anything useful.

On the other hand, we can change implementation a bit (create another issue to implement a new feature) - show only, for example maximum 30 tags with highest number of sources. So, for example, initially minoritary tags, like ```Log Viewer``` and ```Log Writer``` will not be visible at all. When I select ```Logging``` and results (and thus, tags in cloud) will be filtered, they will naturally appear.

Reply:
Agree, I will remove `Log Viewer`, and keep `Logging`